Assets/Scripts/Buildings/Campfire.cs

```diff
@@ -0,0 +1,35 @@
+using Constants;
+using Entity.Player;
+using UnityEngine;
+
+namespace Buildings
+{
+    [RequireComponent(typeof(Collider))]
+    public class Campfire : MonoBehaviour
+    {
+        [Tooltip("The amount of damage the fire deals to the player")] [SerializeField]
+        private int damage;
+
+        [Tooltip("The interval in which to apply the damage to the player")] [SerializeField]
+        private float damageInterval;
+
+        private float _timer;
+
+        private void OnTriggerStay(Collider other)
+        {
+            if (!other.tag.Equals(Consts.Tags.PLAYER)) return;
```

Comment:
```suggestion
            if (!other.CompareTag(Consts.Tags.PLAYER)) return;
```
If you do it with OnTriggerStay, then use this. It's a bit faster:P